### ISSUE TITLE
triple icons after update

### ISSUE THREAD

**Issue by r-luna** [DH3ALEJANDRO/custom-work-for-brackets#36]
The file tree now shows three icons after the 5.0.9 update. I had Brackets Icons 1.5.6 installed, disabling it resolved the issue. Re-enabling Brackets Icons didn't have an effect - single icons, no double or triple. I've removed Brackets Icons from my setup as I suppose I no longer have a need for it...
 
![3icons](https://cloud.githubusercontent.com/assets/6866576/12423690/b0a60068-be9a-11e5-9b2e-e12e731fa947.png)

**Comment by r-luna** [comment]
I saw this thread:

https://github.com/DH3ALEJANDRO/custom-work-for-brackets/issues/29#issuecomment-170993607

I actually prefer the icons in Brackets Icons instead of yours - anyway that I can use different icons other than the ones you provide or allow Brackets Icons to be the icon provider?

**Comment by r-luna** [comment]
Further testing shows that this setup results in three icons appearing with Brackets Icons enabled and disabled:

    "custom.work.geticonprovider": false,
    "custom.work.addiconprovider": true

Additionally if I change the value of "custom.work.geticonprovider" to true and false multiple times i can get icons to stack up in the file tree column. Note the stacked icons are yours and not the ones from the "Brackets Icons" plugin.

![6icons](https://cloud.githubusercontent.com/assets/6866576/12423232/7d750632-be98-11e5-9ec7-3d55179b7398.png)

Seems that with this config i get what i want - your icons in your tabs and the icons from "Brackets Icons" in my file tree.

    "custom.work.geticonprovider": false,
    "custom.work.addiconprovider": false 

![desired_icons](https://cloud.githubusercontent.com/assets/6866576/12423303/d3979174-be98-11e5-97e1-485af2f503ba.png)

**Comment by DH3ALEJANDRO** [comment]
it's solved :smiley_cat:

**Comment by patorpey** [comment]
In 5.0.9 with Brackets Icons, using
```
"custom.work.geticonprovider": true,
"custom.work.addiconprovider": false
```
I was able to get the Brackets Icons icons in both the file tree and in tabs, without any duplicate icons, which was exactly the behavior I was looking for.

With the update to 5.1.0, I now see Brackets Icons icons in the file tree, but the default icon set for Custom Work Brackets in the tabs, i.e. two different icon sets as pictured in the final image of r-luna's post above. The `geticonprovider` option appears to be removed from the Custom Work for Brackets preferences UI in 5.1.0, though `"custom.work.geticonprovider": true` is still set in my Brackets preferences. For me, having a single active icon set, the one I've selected using Brackets Icons as I was able to achieve in 5.0.9, would be desirable.
